### PR TITLE
feat: User(Admin) information page and Admin Updating page

### PR DESCRIPTION
**Pull request by Davyyy**
Connecting with Mock API, updating (put) passed network -200.

### CODE REVIEW

File: website/src/pages/AdminUpdate/admin.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.ant-upload.ant-upload-select{
+/*     width: 50% !important;
+    height: auto  !important; */
+    padding:40%;
+}
+
+.avator{
```
Comment (StevenDB995) [inline website/src/pages/AdminUpdate/admin.css:7]: typo: avatar

File: website/src/pages/AdminUpdate/index.tsx [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
+  };
+
+  const handleFormSubmit = () => {
+    console.log('click');
```
Comment (StevenDB995) [inline website/src/pages/AdminUpdate/index.tsx:68]: Please avoid using console.log. Delete them after testing

File: website/src/pages/AdminUpdate/index.tsx [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+  const fetchData = async () => {
+    try {
+      const response = await fetch('https://my.api.mockaroo.com/data.json?key=e7cb8c20'); // Replace with your API endpoint
+      const jsonData = await response.json();
+      setData(jsonData);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  }; */
+  //////////////////
```
Comment (StevenDB995) [inline website/src/pages/AdminUpdate/index.tsx:135]: Remove commented code segament. Keep your code clean

File: website/src/pages/AdminView/adminView.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    font-size: 2vw;   
+    text-align: left; 
+}
+.container{
+    margin-top: 6vh;
+    margin-left: 10vh;
+/* top:500px;
+position: relative; */
+}
+.avator{
```
Comment (StevenDB995) [inline website/src/pages/AdminView/adminView.css:21]: typo

**Review by StevenDB995 — CHANGES_REQUESTED** [review]

**Review by StevenDB995 — APPROVED** [review]

File: website/src/pages/AdminView/index.tsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+export default function index() {
+  //////////
+  const navigate = useNavigate();
+  const handleNavigation = () => {
+    navigate('/adminUpdate');
+  };
+  ////////////
+
+  const url = 'https://my.api.mockaroo.com/data.json?key=e7cb8c20';
```
Comment (liuxt805) [inline website/src/pages/AdminView/index.tsx:15]: what is this url? shouldn't it be something like "/api/admin/users/:id"?

File: website/src/pages/AdminUpdate/index.tsx [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+  const success = () => {
+    messageApi.open({
+      type: 'success',
+      content: 'Upload success',
+    });
+  };
+
+  /////////////// API
+  const url = 'https://my.api.mockaroo.com/data.json?key=e7cb8c20';
+  const posturl = 'https://my.api.mockaroo.com/data2.json?key=e7cb8c20&__method=PUT';
```
Comment (liuxt805) [inline website/src/pages/AdminUpdate/index.tsx:43]: external api?

Comment (liuxt805) [reply website/src/pages/AdminUpdate/index.tsx:43]: please implement mock api as well :))

**Review by liuxt805 — APPROVED** [review]

**Pull request merged**